src/main/resources/archetype-resources/__rootArtifactId__-module/src/main/resources/xsl/actionsAndDerivates.xsl

```diff
@@ -171,14 +198,14 @@
         </div>
       </div>
 
-      <div id="mcr-collapse-{@xlink:href}" class="row body collapse in">
+      <div id="mcr-collapse-{@xlink:href}" class="row body collapse show">
         <xsl:variable name="ifsDirectory" select="document(concat('ifs:',$derId,'/'))" />
         <xsl:variable name="numOfFiles" select="count($ifsDirectory/mcr_directory/children/child)" />
         <xsl:variable name="maindoc" select="$derivateXML/mycorederivate/derivate/internals/internal/@maindoc" />
         <xsl:variable name="path" select="$ifsDirectory/mcr_directory/path" />
 		
 		<xsl:if test="acl:checkPermission($derId,'writedb')">
-  		 <div id="mcr-file-upload-{$derId}" class="file-upload-box well well-sm collapse col-xs-10 col-xs-offset-1" style="margin-top:1em" data-upload-object="{$derId}" data-upload-target="/">
+  		 <div id="mcr-file-upload-{$derId}" class="file-upload-box well well-sm collapse col-10 col-offset-1" style="margin-top:1em" data-upload-object="{$derId}" data-upload-target="/">
```

Comment:
Bootstrap 4 replaced well with cards
https://getbootstrap.com/docs/4.0/migration/#panels-thumbnails-and-wells